plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -310,28 +416,22 @@ fu_pxi_device_wait_notify (FuPxiDevice *self,
 	}
 	/* get the opcode if status is not null */
 	if (status != NULL) {
-		guint8 status_tmp = 0x0;
 		if (!fu_common_read_uint8_safe (res, sizeof(res), 0x1,
-						&status_tmp, error))
+						status, error))
```

Comment:
i added all this for a reason in the last commit

Reply:
I think it's because they didn't rebase against master and just did a diff against their master somehow. Working in a branch would have prevented this.